Add an Environmental Variable to Define Resources Path. (#233)
omichel/webots: docs/guide/preferences.md

It should work on most systems assuming that `python` is installed and available from the command line.
On some systems, it may be useful to set it to `python3.7` for example if you want to launch the controllers with this specific version of Python.
Bear in mind that this value may be overridden by the content of a `runtime.ini` file of a Python controller that may redefine a specific Python command to launch that controller.

> - The **Extra projects path** can be used to define the path to a folder containing extra projects resources that can be used in the current project (such as PROTO nodes, controllers, textures, etc.).

Reviewer: ```suggestion
- The **Extra projects path** defines the path to a user folder similar to the `WEBOTS_HOME/projects` folder.
This user folder should contain projects resources that can be used in the current project (such as PROTO nodes, controllers, textures, etc.).
```

> This folder can contain multiple sub-folders, each one associated to one sub-project (which should respect the [Standard File Hierarchy of a Project](the-standard-file-hierarchy-of-a-project.md)).

Reviewer: ```suggestion
It may contain multiple sub-folders, each one associated to one sub-project (which should respect the [Standard File Hierarchy of a Project](the-standard-file-hierarchy-of-a-project.md)).
```
